Update / refactor the Ohai docs a bit

tas50 opened the pull request.

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

cwebberOps commented:
@delivery approve

chef-delivery commented:
Change: c453aaf3-fe84-4e8c-92da-e8a7035dccdc approved by: @cwebberOps

chef-delivery commented:
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

tas50 committed:
Remove the ohai summary from the ohai ctl page

Redundant information here. We have a simpler description below.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
We’ve never supported non-NT Windows and there’s no need to note it

No one is trying to run Chef on Windows ME

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Simplify the intro to the custom ohai plugins page

Leave out the summary and just link to the ohai page instead

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove the custom plugin information from the main ohai page

We have an entire page for this information. Just link to it.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Update the ohai summary tag

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
This command is awful, but lets at least make it work

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove the attribute persistence section

We're diving heavy into attributes here when this is the Ohai page.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Fix sections to be properly nested

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove more custom ohai plugin documentation

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Better describe why / how to use the ohai resource

Give the example people actually want

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Move / update the Ohai 6 plugin migration docs

We link to this from our deprecation pages now. Update this doc to reflect the fact that we've deprecated v6 plugins

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Revert "Better describe why / how to use the ohai resource"

This reverts commit 06b9994a1145958dcd6416f9a2b272d3075fabae.

thommay commented on chef_master/source/chef_client.rst:
```diff
@@ -57,20 +57,20 @@ The key components of nodes that are under management by Chef include:
-       * Memory usage
-       * CPU data
-       * Kernel data
+       * Platform
```
As a casual reader, I can't really tell what `platform` might mean here. I think this bullet might be better written as "Operating System details"

tas50 replied:
Changed to Operating System

thommay commented on chef_master/source/ohai.rst:
```diff
@@ -5,26 +5,26 @@ About Ohai
 .. tag ohai_summary
 
-Ohai is a tool that is used to detect attributes on a node, and then provide these attributes to the chef-client at the start of every chef-client run. Ohai is required by the chef-client and must be present on a node. (Ohai is installed on a node as part of the chef-client install process.)
+Ohai is a tool that is used to collect system configuration data, which is provided to the chef-client for use within cookbooks. Ohai is run by the chef-client at the beginning of every Chef run to determine system state. It includes multiple built in plugins for configuration detection and a plugin model for writing custom plugins.
```
"includes many built in plugins" ?

tas50 replied:
The updated wording work for you?

thommay commented on chef_master/source/ohai_custom.rst:
```diff
@@ -434,7 +410,7 @@ and so on, for each of the various cloud providers.
 .. end_tag
 
-Log Entries
+Loggging
```
i like your commitment to the letter 'g' here

tas50 replied:
We loggg a lot

Commit:
Minor copyedit

Signed-off-by: David Wrede <dwrede@chef.io>

tas50 committed:
Update the list of plugins we ship

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Reword the plugins sentence

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 commented:
@davidwrede This is ready for merge now

kagarmoe commented:
![tenor-12262506](https://cloud.githubusercontent.com/assets/4400151/26016051/538535aa-3718-11e7-914c-99360e0ebc47.gif)
